packages/docker-registry-certs/packaging

```diff
@@ -0,0 +1,17 @@
+set -eux
+
+source /var/vcap/packages/golang-1.12-linux/bosh/compile.env
+
+mkdir -p ${BOSH_COMPILE_TARGET}/src/go-src
+mv ${BOSH_COMPILE_TARGET}/go-src/docker-registry-certs ${BOSH_COMPILE_TARGET}/src/go-src/docker-registry-certs
+mv ${BOSH_COMPILE_TARGET}/go-src/vendor ${BOSH_COMPILE_TARGET}/src/go-src/vendor
+
+export GOPATH=${BOSH_COMPILE_TARGET}
+
+# Run unit tests
```

Comment:
Remove the unit tests from the packaging script.

Reply:
Done!